tests/silent-payment.test.ts

```diff
@@ -1,24 +1,42 @@
 import assert from "node:assert";
 import { ECPairFactory } from "ecpair";
-import { SilentPayment } from "../src";
+import { SilentPayment, UTXOType } from "../src";
 import ecc from "../src/noble_ecc";
-import jsonImput from "./data/sending_test_vectors.json";
+import { Vin, getUTXOType } from "../tests/utils";
+import jsonInput from "./data/sending_test_vectors.json";
 
 const ECPair = ECPairFactory(ecc);
 
-type TestCase = {
-  comment: string;
-  given: {
-    outpoints: [string, number][],
-    input_priv_keys: [string, boolean][],
-    recipients: [string, number][]
-  },
-  expected: {
-    outputs: [string, number][],
-  }
+function exactMatch(a: string[], b: string[]): boolean {
+    const sortedA = a.sort();
+    const sortedB = b.sort();
+    return sortedA.length === sortedB.length && sortedA.every((value, index) => value === sortedB[index]);
+}
```

Comment:
will `assert.deepStrictEqual()` work for you..?

Reply:
We have a generated set of outputs `G` and I want to know if it matches one of the possible output sets in expected, `E = [A, B, C]`. I don't think `assert` will work because it will fail the whole test as soon as `G` doesn't match one of the output sets, but what I am checking for is that `G` matches exactly one of the output sets in `E`.